### PR TITLE
Update readme to reflect recent changes

### PR DESCRIPTION
**Pull request by cc-a**
# Description

Our development setup has evolved somewhat recently and the README colud use a refresher to provide a smoother onboarding experience.

Fixes # (issue)

## Type of change

- [X] Documentation (non-breaking change that adds or improves the documentation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (non-breaking, back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (whatever its nature)

## Key checklist

- [ ] All tests pass (eg. `python -m pytest`)
- [ ] The documentation builds and looks OK (eg. `python -m sphinx -b html docs docs/build`)
- [ ] Pre-commit hooks run successfully (eg. `pre-commit run --all-files`)

## Further checks

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added or an issue has been opened to tackle that in the future. (Indicate issue here: # (issue))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,45 @@
...
+## Running the App
+
+To run with a demo version of the drunc process manager, run it with docker compose:
+
+```bash
+docker compose up
+```
+
+It can take a few moment for the services to boot but the application should then be
+available in the browser at <http://localhost:8000>.  Authentication is required to work
```
Comment (dalonsoa) [inline README.md:20]: The link checker is never happy with `localhost`. This needs to be exlcuded.

**Review by dalonsoa — APPROVED** [review]
Aside from the broken link, this looks good.

File: README.md (hunk 1 above)
Comment (cc-a) [reply README.md:20]: Ugh, well that should be the default behaviour... I've added a config file to ignore all references to it.

**Review by alexdewar — APPROVED** [review]
LGTM :+1:

**Review by jamesturner246 — COMMENTED** [review]
Looks good. :+1:  Seems to have all bases covered. Nothing to add.

**Pull request merged**